src/components/custom-labels-section/custom-labels-section.js

```diff
@@ -1,42 +1,17 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import CustomLabelsEditSection from './custom-labels-edit-section';
+import CustomLabelsShowSection from './custom-labels-show-section';
 
-import {
-  KeyValue,
-  NoValue,
-} from '@folio/stripes/components';
-
-import KeyValueColumns from '../key-value-columns';
-
-export default class CustomLabelsSection extends React.Component {
-  static propTypes = {
-    customLabels: PropTypes.array.isRequired,
-    userDefinedFields: PropTypes.object.isRequired,
-  }
-
-  render() {
-    const {
-      customLabels,
-      userDefinedFields,
-    } = this.props;
-
-    return (
-      <KeyValueColumns>
-        {
-          customLabels.map(({ attributes }) => {
-            const value = userDefinedFields[`userDefinedField${attributes.id}`];
-
-            return (
-              <KeyValue
-                data-test-eholdings-resource-custom-label
-                key={attributes.id}
-                label={attributes.displayLabel}
-                value={value || <NoValue />}
-              />
-            );
-          })
-        }
-      </KeyValueColumns>
-    );
-  }
+export default function CustomLabelsSection({ editMode, ...rest }) {
+  return (editMode
+    ? <CustomLabelsEditSection {...rest} />
+    : <CustomLabelsShowSection {...rest} />
+  );
 }
+
+CustomLabelsSection.propTypes = {
+  customLabels: PropTypes.array.isRequired,
+  editMode: PropTypes.bool,
+  userDefinedFields: PropTypes.object.isRequired,
```

Comment:
Please, use `propTypes.shape`  to describe component's props. Also, it would be great to move shapes that are used more than once  to a separate file, like `/shapes/customLabels`, `/shapes/userDefinedFields`